siriuspy/siriuspy/search/id_search.py

```diff
@@ -0,0 +1,99 @@
+"""ID Search module."""
+
+from threading import Lock as _Lock
+
+from .. import clientweb as _web
+from ..magnet.excdata import ExcitationData as _ExcitationData
+
+from .ps_search import PSSearch as _PSSearch
+
+
+class IDSearch:
+    """ID Search Class."""
+
+    _beamline2subsection = {
+        'Carnauba': '06SB',
+        'Caterete': '07SP',
+        'Ema': '08SB',
+        'Manaca': '09SA',
+        'Sabia': '10SB',
+        'Ipe': '11SP',
+        'PGM++': '12SB',
```

Comment:
wouldn't it be better to use `PGMpp`

Reply:
In fact, I believe the right name is "COLIBRI"! 
https://wiki-sirius.lnls.br/mediawiki/index.php/Table:Storage_ring_straight_sections_allocation

 I will fix this in the next commit!

Reply:
Ok, @anacso17 !